data/tlg0732/tlg012/tlg0732.tlg012.1st1K-grc1.xml

```diff
@@ -0,0 +1,6021 @@
+                    <resp>Corrected and encoded the text</resp>
+                </respStmt>
+                <respStmt>
+                    <persName>Gregory Crane</persName>
+                    <resp>Editor-in-Chief, Perseus Digital Library</resp>
+                </respStmt>
+                <respStmt>
+                    <persName>Matt Munson</persName>
+                    <resp>Project Manager (University of Leipzig), 2016 - present</resp>
+                </respStmt>
+                <respStmt>
+                    <persName>Annette Gessner</persName>
+                    <resp>Project Assistant (University of Leipzig) 2015 - present</resp>
+                </respStmt>
+                <respStmt>
+                    <persName>Thibault Clérice</persName>
+                    <resp>Lead Developer (University of Leipzig)</resp>
+                </respStmt>
+                <respStmt>
+                    <persName>Bruce Robertson</persName>
+                    <resp>Technical Advisor</resp>
+                </respStmt>
+            </titleStmt>
+            <publicationStmt>
+                <authority>Harvard College Library</authority>
+                <idno type="filename">tlg0732.tlg012.1st1K-grc1.xml</idno>
+                <availability>
+                    <p>Available under a Creative Commons Attribution-ShareAlike 4.0 International
+                        License</p>
+                </availability>
+                <date>2017</date>
+                <publisher>Harvard College Library</publisher>
+                <pubPlace>United State</pubPlace>
+            </publicationStmt>
+            <sourceDesc>
+                <listBibl>
+                    <biblStruct>
+                        <monogr>
+                            <title xml:lang="lat">Supplementum Aristotelicum</title>
+                            <editor>
+                                <persName>
+                                    <name>Ivo Bruns</name>
+                                </persName>
+                            </editor>
+                            <author ref="urn:cts:greekLit:tlg0732">Alexander of Aphrodisias</author>
+                            <imprint>
+                                <publisher>Reimer</publisher>
+                                <pubPlace>Berlin</pubPlace>
+                                <date>1887</date>
+                            </imprint>
+                            <biblScope unit="volume">2</biblScope>
+                        </monogr>
+                        <ref target="http://www.archive.org/stream/p1supplementumar02akaduoft#page/n301/mode/2up">Internet Archive</ref>
+                    </biblStruct>
+                </listBibl>
+            </sourceDesc>
+        </fileDesc>
+        <encodingDesc>
+            <p>Text encoded in accordance with the latest EpiDoc standards (January 2014)</p>
+            <refsDecl n="CTS">
+                <cRefPattern matchPattern="(.+).(.+).(.+)" n="section" 
```

Comment:
It looks like you only have "section"s under one of the chapters. I would suggest, to keep the citation scheme the same, that you also add "section"s under the other chapters, even if it means that most of them only have a single section. Then all words will be at the same citation level.